Resources/Prototypes/_NF/Catalog/Fills/Crates/chemistry.yml

```
@@ -8,3 +8,28 @@
       contents:
         - id: JugSpaceCleaner
           amount: 5
+
+- type: entity
+  id: CrateChemistryDL
+  parent: CrateChemistrySecure
+  name: chemicals crate (DL)
+  description: Contains chemicals from the bargain bin overstock-block of elements. Requires understanding of hinges to open.
```

DL... hmmm... hmmmmmmm...

Jokes aside, might honestly just make it say something like this.  Not verbatim, but something along these lines.
```suggestion
  name: chemicals crate (uncommon)
  description: Contains chemicals that rarely occur in nature. Requires understanding of hinges to open.
```